docs/user/component-catalog.md

```
@@ -65,3 +65,57 @@ The output lists every component with its pinned version and configuration value
+### `gpu-operator`: `dcgm-exporter` ConfigMap moved into the main release
+
+Earlier bundles shipped the `dcgm-exporter` ConfigMap as a post-manifest in a separate Helm release named `gpu-operator-post`. The in-cluster ConfigMap therefore carries ownership annotations pointing at that release:
+
+```
+meta.helm.sh/release-name: gpu-operator-post
+meta.helm.sh/release-namespace: gpu-operator
+```
+
+Newer bundles render the ConfigMap directly from the main `gpu-operator` chart's `dcgmExporter.config.data` values. On upgrade, Helm 3 refuses to claim the existing ConfigMap because its annotations point at a different release:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)